resources/js/components/Theme.vue

```diff
@@ -0,0 +1,71 @@
+<script>
+import Vue from 'vue';
+
+const moment = require('moment');
+require('moment/locale/de');
+
+Vue.use(require('vue-moment'), {
+    moment
+});
+
+export default {
+    name: 'Theme',
+    props: {
+        themeListData: {
+            type: Array,
+            default: () => ([])
+
+        },
+        themeListType: {
+            type: String,
+            default: ''
+        },
+    },
+};
+</script>
+
+<template>
+    <div>
+        <a class="ris-card-list__item"
+            v-for="{ id, name } in themeListData"
+            :key="id"
+            :href="'/thema/' + id"
+            :title="name"
+                >
+            <div class="ris-card-list__themes-top">
+                <img src="/img/thumbnail-map-tile.png" class="ris-card-list__themes-img"
+                    :alt="name">
+                <div class="ris-body-1">
+                    {{ name }}
+                </div>
+            </div>
+            <div class="ris-card-list__themes-bottom">
+                <div class="ris-caption ris-card-list__themes-number">
+                    Thema &nbsp;
+                    <span>2477</span>
+                    /
+                    <span>2018</span>
+                </div>
+
+                <div class="ris-progress-bar"
```

Comment:
to avoid code duplication i guess better to build logic like this:
```if (finished) {
finished html
} else  {
everything else (progress bar)
}
```